### DISCUSSION TITLE
eslint-plugin-lodash-f "bug"

### DISCUSSION THREAD

**Discussion by lishaduck** [AndreaPontrandolfo/eslint-plugin-lodash#1] [General]
For lack of an issues page there, posting here (can you add an issues page?):
eslint-plugin-lodash-f rules link to the original eslint-plugin-lodash.
Additionally, the repo is still named eslint-plugin-lodash. Would be nice if that got updated, though I recognize that you use remeda now. However, you don't here yet, so still filing.
Found in #162.

**Comment by AndreaPontrandolfo** [comment]
Hey @lishaduck !

So, i changed the name of the repo to [eslint-plugin-lodash-f](https://github.com/AndreaPontrandolfo/eslint-plugin-lodash-f) and added the Issues page!
However i don't understand what you mean when you say:

> eslint-plugin-lodash-f rules link to the original eslint-plugin-lodash

Can you clarify what you mean?

**Reply by lishaduck** [reply]
In VS Code, it includes a link to the rules' documentation, but the links are to lodash, I think you need to edit this line: https://github.com/AndreaPontrandolfo/eslint-plugin-lodash-f/blob/cee83f7dcff715d5f0ad8433eedf31404c5a1b0f/src/util/getDocsUrl.js#L5

Also, feel free to move this to the `lodash-f` issues page.

**Reply by AndreaPontrandolfo** [reply]
[Done](https://github.com/AndreaPontrandolfo/eslint-plugin-lodash-f/issues/2)
